tests/test_scripts.py

```diff
@@ -1,40 +1,50 @@
-"""This is a sample python file for testing scripts from the source code."""
+"""Tests for the scripts in the scripts directory."""
 
 from __future__ import annotations
 
-import subprocess
+import sys
+from pathlib import Path
 
-import pysam
+# Add the scripts directory to the Python path
+scripts_dir = Path(__file__).resolve().parent.parent / "scripts"
+sys.path.insert(0, str(scripts_dir))
 
+# Import the process_directory function from vp_transformer.py
+from vp_transformer import get_metadata  # noqa: E402 # pyright: ignore
+from vp_transformer import process_directory  # noqa: E402 # pyright: ignore
```

Comment:
In case, `pyright` is killing you. That's how you suppress `pyright` and `ruff`.  ;)